releasenotes/notes/add-more-postgres-exec-plans-adcfd187630acd5d.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Reveals safe and useful information within a SQL execution plan for Postgres.
```

Comment:
nit: let's remove "safe"

I think it gives the wrong impression as anything we choose to expose is implicitly deemed safe.